docs/get-involved/community-garden.md

```diff
@@ -0,0 +1,19 @@
+---
+id: radicle-garden
+title: Radicle Garden 🌞
+---
+
+The Radicle Garden features contributions from our community. If you've built
+something cool on Radicle, [submit it here][gh] 🌱
+
+→ [radicle-proxy-cli][cli] by @Nemo157 
+* rad:git:hwd1yre85oenojqfpzt1ihd36enmupg4dnb1o78rsos8h957bpxxbzn5g4a 
```

Comment:
We'll have to remember to ask the authors to re-upload these to the new seed node and update these IDs once we release the next breaking change.

Reply:
Yes +1. I can DM them all